_commands/basics/printf.md

```diff
@@ -0,0 +1,99 @@
+---
+---
+
+printf
+--
+
+`printf` is used to format and output data.
+
+~~~ bash
+$ printf format [argument]
+$ printf "%f\n" 7
+7.000000
+$ printf "%d\n " 0xF
+15
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `printf`
+~~~ bash
+$ printf "%f\n" 5
+~~~
+
+##### Break it down
+
+*This is a simple use of 'printf'. It prints out the number 5 as a float on a new line. The default is to print with 6 decimals.
+
+~~~ bash
+$ printf "\t\"Quotations\"\t\n"
+	"Quotations"	
+$
+~~~
+
+##### Break it down
+
+*This command takes advantage of printf's ability to use whitespace by adding tabs into the output using \t to add tabs and a newline at the end
+*This command also demonstrates how to add characters to strings that would normally give a command to printf. Quotations usually specify what is meant to be fed into printf, but using \" allows an actual double quote to be inserted into the string.
```

Comment:
You'll need a newline before this asterisk for it to render on its own line.

Also, printf -> `printf` a couple times in this sentence.

You'll also need \\\\" instead of just one backslash for both the backslash and the quote to be rendered.